e2e/libs/keywords/backupstore_keywords.py

```diff
@@ -1,60 +1,46 @@
-from backupstore import Nfs, Minio
-from utility.utility import get_longhorn_client
+from backupstore import Nfs, S3
 from utility.utility import get_backupstore
-from kubernetes import client
-from setting import Setting
+from utility.utility import logging
+import os
 
 
 class backupstore_keywords:
 
     def __init__(self):
         backupstore = get_backupstore()
         if backupstore.startswith("s3"):
-            self.backupstore = Minio()
+            self.backupstore = S3()
         elif backupstore.startswith("nfs"):
             self.backupstore = Nfs()
-        self.setting = Setting()
-
-    def set_backupstore(self):
-        self.setting.set_backupstore()
-
-    def cleanup_backupstore(self):
-        if get_backupstore():
-            client = get_longhorn_client()
-            self.backupstore.cleanup_system_backups(client)
-            self.backupstore.cleanup_backup_volumes(client)
-            self.setting.reset_backupstore()
 
     def create_dummy_in_progress_backup(self, volume_name):
-        client = get_longhorn_client()
-        core_api = client.CoreV1Api()
 
         dummy_backup_cfg_data = {"Name": "dummy_backup",
                                  "VolumeName": volume_name,
                                  "CreatedTime": ""}
 
-        self.backupstore.write_backup_cfg_file(client,
-                                               core_api,
-                                               volume_name,
+        self.backupstore.write_backup_cfg_file(volume_name,
                                                "backup-dummy",
                                                dummy_backup_cfg_data)
 
     def delete_dummy_in_progress_backup(self, volume_name):
```

Comment:
qq: Why the name is `in_progress`?

Reply:
An _artificial_ _in progress_ backup can be achieved by creating a dummy `backup.cfg` file with empty `CreatedTime`:
```
json.dumps({"Name": name, "VolumeName": volume, "CreatedTime": ""})
```
This function is ported by the original `backupstore.py`, and the corresponding robot test case hasn't been implemented yet. If there's any confusing, it can be refined in the future.